frontend/test/__support__/e2e/helpers/e2e-misc-helpers.js

```diff
@@ -132,6 +132,17 @@ export function visitQuestion(id) {
   cy.wait("@" + alias);
 }
 
+/**
+ * Visit an archived question.
+ *
+ * @param {number} id
+ */
+export function visitArchivedQuestion(id) {
+  cy.visit(`/question/${id}`);
+
+  cy.findByText("This question has been archived");
+}
+
```

Comment:
Let's, **please** avoid creating global helpers for something that's going to be used in just a couple of tests.

Side note, there is already a helper called `visitQuestion(:id)` that "intelligently" waits for the necessary queries to finish loading. Use that rather than `cy.visit(/question/${id});`

Reply:
Thanks @nemanjaglumac 

Thing with `visitQuestion` for this test is that it waits for something to load that will not be loaded in an archived question's page.

We will have 2 uses of this check for an archived question's page.
If you prefer not to have a dedicated function doing it, no problem, I'll inline the code in each test.

Reply:
Oh, right.
That is an excellent opportunity and a reminder for me to add that logic to that helper.